.github/workflows/enos-test-ui.yml

```diff
@@ -0,0 +1,107 @@
+        run: |
+          echo "go-version=$(cat ./.go-version)" >> $GITHUB_OUTPUT
+          if ${IS_ENT} == true; then
+            echo "vault_edition=ent" >> $GITHUB_OUTPUT
+            echo "detected vault_edition=ent"
+          else
+            echo "vault_edition=oss" >> $GITHUB_OUTPUT
+            echo "detected vault_edition=oss"
+          fi
+      - name: Set Up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ steps.get-metadata.outputs.go-version }}
+      - name: Configure AWS credentials from Test account
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_CI }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_CI }}
+          aws-region: us-east-1
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN_CI }}
+          role-skip-session-tagging: true
+          role-duration-seconds: 3600
+      - uses: hashicorp/action-setup-enos@v1
+        with:
+          github-token: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
+      - name: Set Up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Set Up Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
+          terraform_wrapper: false
+      - name: Prepare scenario dependencies
+        run: |
+          mkdir -p ./enos/support/terraform-plugin-cache
+          echo "${{ secrets.SSH_KEY_PRIVATE_CI }}" > ./enos/support/private_key.pem
+          chmod 600 ./enos/support/private_key.pem
+      - name: Set Up Vault Enterprise License
+        if: contains(${{ github.event.repository.name }}, 'ent')
+        run: echo "${{ secrets.VAULT_LICENSE }}" > ./enos/support/vault.hclic || true
+      - name: Setup Cluster
+        id: setup_cluster
+        env:
+          ENOS_VAR_ui_run_tests: false
+        # Continue once and retry to handle occasional blips when creating infrastructure.
+        continue-on-error: true
+        run: enos scenario launch --timeout 60m0s --chdir ./enos ui edition:${{ steps.get-metadata.outputs.vault_edition }} backend:${{ inputs.storage_backend }}
+      - name: Retry Setup Cluster
+        id: setup_cluster_retry
+        if: steps.setup_cluster.outcome == 'failure'
+        env:
+          ENOS_VAR_ui_run_tests: false
+        run: enos scenario launch --timeout 60m0s --chdir ./enos ui edition:${{ steps.get-metadata.outputs.vault_edition }} backend:${{ inputs.storage_backend }}
+      - name: Run UI Tests
+        id: run_ui_tests
+        run: enos scenario run --timeout 60m0s --chdir ./enos ui edition:${{ steps.get-metadata.outputs.vault_edition }} backend:${{ inputs.storage_backend }}
+      - name: Ensure scenario has been destroyed
+        if: ${{ always() }}
+        # With Enos version 0.0.11 the destroy step returns an error if the infrastructure
```

Comment:
We can remove this comment as 0.0.16 is what you'll get by default now. Whether or not deleting should fail the job is a different story I guess. I think it's fine to let it proceed on failure as we have the reaper workflow in place.